plugins/gifd_d4_rules.js

```diff
@@ -2647,16 +2667,18 @@ gifd.rules = {
        ['M_ATN_1Q100','0','raw:M_ATN_1Q100','unknown'], // Unknown
        ['M_ATN_1Q100','1','raw:M_ATN_1Q100','marked'], // Marked
        ['M_ATN_1Q100','2','raw:M_ATN_1Q100','unmarked'], // Unmarked
+       // ['M_ATN_1Q100','998',undefined,undefined], // Not Applicable
+       ['M_ATN_1Q100','998','raw:M_ATN_1Q100','not_applicable'], // Not Applicable
```

Comment:
Just a nit-pick, do you want these in order?  You inserted 998 in front of 997.

Reply:
Fixed.